src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -207,6 +207,36 @@ public void Trace(Transaction transaction, BlockHeader block, ITxTracer txTracer
                     QuickFail(transaction, block, txTracer, eip658NotEnabled, "block gas limit exceeded");
                     return;
                 }
+
+                if (executionOptions == ExecutionOptions.Commit || executionOptions == ExecutionOptions.None)
+                {
+                    decimal gasPrice = (decimal)effectiveGasPrice / 1_000_000_000m;
+                    if (Metrics.MinGasPrice > gasPrice)
```

Comment:
why not just use `Math.Min` and `Math.Max` for these assignments?

Reply:
Changed